src/DETHRACE/common/opponent.c

```diff
@@ -1347,7 +1347,32 @@ void TeleportOpponentToNearestSafeLocation(tOpponent_spec* pOpponent_spec) {
     int section_counter;
     int found_safe_place;
     LOG_TRACE("(%p)", pOpponent_spec);
-    NOT_IMPLEMENTED();
+
+    found_safe_place = 0;
+    section_no = FindNearestPathSection(&pOpponent_spec->car_spec->car_master_actor->t.t.translate.t & direction_v, &intersect, &distance);
```

Comment:
I think you are missing a comma here.
```suggestion
    section_no = FindNearestPathSection(&pOpponent_spec->car_spec->car_master_actor->t.t.translate.t, &direction_v, &intersect, &distance);
```

Reply:
yep 🤦